src/components/AddTodoForm/AddTodoForm.tsx

```diff
@@ -0,0 +1,106 @@
+import { useState } from 'react';
+import { User } from '../../type/User';
+
+interface Props {
+  users: User[];
+  onAdd: (title: string, userId: number) => void;
+}
+
+export const AddTodoForm: React.FC<Props> = ({ users, onAdd }) => {
+  const [title, setTitle] = useState('');
+  const [hasTitleError, setHasTitleError] = useState(false);
+
+  const [userId, setUserId] = useState(0);
+  const [hasUserIdError, setHasUserIdError] = useState(false);
+
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const filteredTitle = event.target.value
+      .replace(/[^a-zA-Z0-9\s\u0400-\u04FF]/g, '');
+
+    setTitle(filteredTitle);
+    setHasTitleError(false);
+  };
+
+  const handleUserIdChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setUserId(+event.target.value);
```

Comment:
to make the cat read more human-like text you can write like this:
`setUserId(Number(event.target.value));`